docs/developer_guide.md

```diff
@@ -51,7 +51,32 @@ select **MUXControlsTestApp** as your startup project in Visual Studio.
 
 ### Creating a NuGet package
 
-> More information will be coming on this soon
+To create a NuGet package for a given build flavor (release/debug) and build arch (x64/x86/ARM/ARM64), first you need to build the solution in that configuration.
+
+After building the solution in the desired configuration, you can run the `build-nupkg.ps1` script that will run the required steps.
+The script takes the following arguments:
+- BuildOutput: The path to the microsoft-ui-xaml BuildOutput folder
+- BuildFlavor (default: `release`): The flavor to use for nuget build (`debug` or `release`)
+- BuildArch (default `x86`): The build arch to use, one of: `x64`, `x86`, `ARM`, `ARM64`
```

Comment:
For BuildArch, we will package whichever architectures are available in the BuildOutput, not just what you specify. The value of the param really controls where to get the .winmd and generic.xaml, etc. So BuildArch must be set to one of the archs that you built, but the output will include files from all of the archs that you built. It might be worth clarifying that here.

Reply:
I did not know that, thanks for the correction. Updated the doc accordingly.